### ISSUE TITLE
Gatekeeper reports this app as a virus.

### ISSUE THREAD

**Issue by blackxfiied** [PhoenixLauncher/Phoenix#60]
<img width="1512" alt="Screenshot 2023-09-08 at 14 58 03" src="https://github.com/PhoenixLauncher/Phoenix/assets/41133734/1eaf2a77-f123-4c1c-a26b-ad62b6056c57">

**Comment by Shock9616** [comment]
That's weird, I haven't seen it get flagged as a virus before, but I promise It's not, that is a false positive. If you are worried about it though, you can have a look through the source code and try building it for yourself since it's open source! Feel free to ask any questions you have about the code!

**Comment by blackxfiied** [comment]
yeah i know

just thought i'd let you know so you don't end up with other people having the same problem

what could the cause be?

even running `xattr` doesn't allow for the app to open

**Comment by jxhug** [comment]
1.	Check Notarization Status: Ensure that your app is properly notarized. You should have notarized your app with Apple before distributing it. You can check the notarization status using the spctl command in the Terminal.
	2.	Update Certificates: Ensure that your code signing certificates are up to date and valid. Expired certificates can cause this issue.
	3.	Check App Signing: Make sure your app is correctly signed with your Developer ID certificate.
	4.	Rebuild and Re-Notarize: If everything appears to be in order, try rebuilding your app and notarizing it again.

I've found a few steps to help fix this. I will try to test this soon

**Comment by tqoo** [comment]
Thought I'd chime in, I'm receiving the error "Phoenix can't be opened. You should move it to the Bin" on macOS Sonoma

**Comment by jxhug** [comment]
Alright thanks, will look into this too

**Comment by jxhug** [comment]
<img width="453" alt="SCR-20230908-pigp" src="https://github.com/PhoenixLauncher/Phoenix/assets/72482214/559e2679-9eec-45c9-8123-037909b0bba6">

just got the same error on my build that was working fine, weird

**Comment by jxhug** [comment]
@tqoo @blackxfiied sorry to bother you guys, if you could give the latest release a try it'd be appreciated!

**Comment by jxhug** [comment]
i don't think this happens anymore so i'll close it. may be reopened in the future
